emsdk.py

```diff
@@ -2842,6 +2842,18 @@ def expand_sdk_name(name, activating):
 
 
 def main(args):
+
+  if is_emsdk_sourced_from_github():
+    # This code only exists on the master branch
+    errlog('****')
+    errlog('Warning: You appear to be using the `master` branch of emsdk.')
+    errlog('We recently made the switch to using `main`')
+    errlog('In order to continue to receive updates will need to make the switch locally too.')
+    errlog('For trivial checkouts simply running the following command should be enough:')
+    errlog('  `git checkout main`')
+    errlog('For more complex use cases see https://github.com/emscripten-core/emsdk/issues/805')
+    errlog('****')
```

Comment:
I don't follow these lines. What is a "trivial checkout"? And more complex cases do not seem to be mentioned in that link?

Reply:
I meant if you don't have any local branches .. I can clarity.